.travis.yml

```diff
@@ -44,8 +44,6 @@ matrix:
     - os: linux
       env: PROJECT_DIR=examples/librtmp TOOLCHAIN=android-ndk-r10e-api-19-armeabi-v7a-neon
     - os: linux
-      env: PROJECT_DIR=examples/librtmp TOOLCHAIN=analyze
```

Comment:
Please comment this lines out instead of removing them completely. Also please add link/notes so if somebody want to investigate the issue can do it easily. Example:
* https://github.com/ingenue/hunter/blob/6c1c47d52d77738366cbfc0c0ebb4c6ff0f4af52/.travis.yml#L256-L258